tollbooth.go

```diff
@@ -159,7 +158,7 @@ func LimitHandler(lmt *limiter.Limiter, next http.Handler) http.Handler {
 			lmt.ExecOnLimitReached(w, r)
 			w.Header().Add("Content-Type", lmt.GetMessageContentType())
 			w.WriteHeader(httpError.StatusCode)
-			w.Write([]byte(httpError.Message))
+			_, _ = w.Write([]byte(httpError.Message))
```

Comment:
Is this necessary?

Reply:
It's an explicit way to tell whoever reads the code "this function returns an error but I don't care about it and discard it".

Reply:
I can revert it if you want, it's just a common convention. In https://github.com/golang/go.git for example:
```
➜  go git:(master) fgrep '_, _ =' . -r | grep -v "_test.go" | wc -l
     168
➜  go git:(master) fgrep '_ =' . -r | grep -v "_test.go" | wc -l
    7315
```

Reply:
Let's do that :)

Reply:
Done, I'll resolve these threads.